package.json

```diff
@@ -1,9 +1,14 @@
 {
 	"private": true,
-	"name": "o-table",
+	"name": "4table",
```

Comment:
```suggestion
	"name": "o-table",
```

Reply:
woops haha, that was from when i was testing the manifest linter. thanks!!!